src/components/execution/AddExecutionForm.js

```diff
@@ -265,7 +281,12 @@ class AddExecutionForm extends Component {
             value={userProvidedFilename}
           />
         </FormControl>
-
+        {parameters && parameters.length > 0 && (
+          <SelectParameters
```

Comment:
(to be clear, I mean that once you select a parameterized algo, the modal pops up, and is then re-accessible via edit parameters)

Reply:
@hagoptaminian Additionally what JC suggested is to have similar creation/addition of resource. Currently we have 3 methods: modal, new page and in-page form. I would suggest to go for a new page each time we want to create a new entity. Therefore, (and since the execution table is merged) we could have this execution form in a separate page and have the parameters in-page as well. Maybe we can have something similar to the "Add Algorithm" page. What do you think @Painguin @hagoptaminian ?